crates/jiter/benches/main.rs

```diff
@@ -1,5 +1,4 @@
-use codspeed_bencher_compat::{benchmark_group, benchmark_main, Bencher};
-use std::hint::black_box;
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
```

Comment:
We should use `codspeed_criterion_compat` here so that the benchmarks continue to be measured in CI.